src/Components/App/App.js

```diff
@@ -1,22 +1,59 @@
 import React, {Component} from 'react';
 import './App.css';
-import fetchPromises from '../../apiCalls'
+import {fetchPromises, fetchStories} from '../../apiCalls'
+import ArticleContainer from '../ArticleContainer/ArticleContainer'
+import styled from 'styled-components'
+
+const AppStyled = styled.div`
+  width: 100%;
+  height: 100%;
+  border: 2px solid blue;
+`
 
 class App extends Component {
   constructor() {
     super()
-    this.state ={}
+    this.state ={
+      newStoryIDs: [],
+      bestStoryIDs: [],
+      topStoryIDs: [],
+      homePageStories: []
+    }
+  }
```

Comment:
At some point we'll want to figure out some error handling if a story returns null - not exactly sure why some stories are returning null, but if it does, we'll want it to go to the next story.

Reply:
Agreed, either going to next available story and/or displaying a message to user to reload.